src/Esfa.Vacancy.Api.Types/Esfa.Vacancy.Api.Types.csproj

```diff
@@ -45,6 +45,7 @@
       <Link>Properties\SharedAssemblyInfo.cs</Link>
     </Compile>
     <Compile Include="Address.cs" />
+    <Compile Include="CreateApprenticeship\WageUnit.cs" />
```

Comment:
This should fall under a namespace that is not called CreateApprenticeship but ends with 'Request'. E.g. `Esfa.Vacancy.Api.Types.Request`. This is because we will be using for other post actions in the future possibly, like for traineeships.

Reply:
Discussion.